catroid/src/test/java/org/catrobat/catroid/test/robolectric/bricks/BrickSpinnerTest.java

```diff
@@ -159,8 +156,8 @@ public static Collection<Object[]> data() {
 
 	private List<String> expectedContent;
 
-	public BrickSpinnerTest(String name, Brick brick, @IdRes int spinnerId, String expectedSelection, List<String> expectedContent) {
-		this.name = name;
+	public BrickSpinnerTest(String name, Brick brick, @IdRes int spinnerId,
```

Comment:
please resolve the two warnings mentioned in the jenkins report (line 159 and 160)